### DISCUSSION TITLE
Inquiry on Global Planner

### DISCUSSION THREAD

**Discussion by FrankBu0616** [prl-mushr/mushr#65] [Q&A]
Hi! Thank you for the great platform! 
I am wondering what global planner you are using. Currently the page for mushr_gp is down. 

Thank you so much!

**Comment by schmittlema** [comment]
Hi Frank,
Right now we don't have a public global planner because we are revamping out autonomy stack. The new autonomy stack (should be available by the end of the year) utilizes [SBPL](https://github.com/sbpl/sbpl) (A* on lattice) specifically a [python wrapped version ](https://github.com/schmittlema/pysbpl). Let me know if you have any other questions!

**Reply by FrankBu0616** [reply]
Got it! Thank you!

**Reply by Lilly-yang** [reply]
Hi Schmittlema,

Is the new autonomy stack available now? If so, where I can find it? Thank you!

Best,

**Reply by schmittlema** [reply]
sorry for the delay, checkout here: https://github.com/prl-mushr/mushr/discussions/85
